xAPI.md

```diff
@@ -507,6 +507,45 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 7.0 xAPI Components
 
+This section explains and shows graphically how different pieces of xAPI can fit together.  
+
+##### Tracking Experiences Through Systems
+
+![Data Flow in xAPI](/xAPIDataFlow.jpg)
+
+Figure 1 shows the tracking of experiences through the data flow of a series of systems.  The cloud in the picture 
+represents a Learning Management System, or LMS.  An LMS, if present, would likely encapsulate everything within 
```

Comment:
Yeah, opening with that does make it sound prescriptive.  Anyone see value in keeping the LMS but sending it to the back as a closing comment?  I do like providing context to meet people where they are, but could also see that a lack of LMS could be a thing spec viewers would want and expect to see.